### PR TITLE
Documentation refactorisation

### LINKED ISSUE #85: Restyle the documentation
Improve readability of the doctoring when coding up the model; minimal change to the sphinx API docs is expected to occur.

### PR DESCRIPTION
**Pull request by I-Bouros**
Closes #85

### CODE REVIEW

File: pyEpiabm/pyEpiabm/utility/distance_metrics.py [hunk 1]
```diff
@@ -34,12 +41,18 @@ def dist_euclid(loc1: typing.Tuple[float, float],
...
+        ----------
+        loc1 : Tuple[float, float]
+            (x,y) coordinates of the first place
+        loc2 : Tuple[float, float]
+            (x,y) coordinates of the second place
+
+        Returns
+        -------
+        float
+            Euclidean distance between the two tuples
```
Comment (KCGallagher) [inline pyEpiabm/pyEpiabm/utility/distance_metrics.py:54]:
```suggestion
            Euclidean distance between the two location tuples
```

File: pyEpiabm/pyEpiabm/utility/distance_metrics.py [hunk 2]
```diff
@@ -53,16 +66,22 @@ def dist_periodic(loc1: typing.Tuple[int, int],
...
+            Number of indices in a row
+        scales : Tuple[float, float]
+            Conversion to global coordinates
+        loc2 : Tuple[int, int]
+            Index location of the second place
+
+        Returns
+        -------
+        float
+            Euclidean distance between the two tuples
```
Comment (KCGallagher) [inline pyEpiabm/pyEpiabm/utility/distance_metrics.py:83]:
```suggestion
            Periodic distance between the two location tuples
```

**Review by KCGallagher — CHANGES_REQUESTED** [review]
Looks great - thank you for doing all this! I realised some of the docstrings were unclear so have updated them, but no changes to your work

File: pyEpiabm/docs/source/conf.py [hunk 3]
```diff
@@ -50,12 +50,15 @@
     autodoc_default_options = {
         'members': None,
         'inherited-members': None,
+        'imported-memebers': None,
```
Comment (KCGallagher) [inline pyEpiabm/docs/source/conf.py:53]:
```suggestion
        'imported-members': None,
```
It seemed to work anyway but I assume this is a typo?

**Review by KCGallagher — COMMENTED** [review]
Looks good - one potential typo!
Also @I-Bouros if you wouldn't mind looking over my docs for `python_examples/spatial_example/voronoi_plotting_example.py` just to have a second eye on them, that would be great!

File: pyEpiabm/docs/source/conf.py (hunk 3 above)
Comment (I-Bouros) [reply pyEpiabm/docs/source/conf.py]: Good catch, @KCGallagher! I forgot to take that out, we don't actually need it

**Review by KCGallagher — APPROVED** [review]
Do look through my last few commits, but I think this is all ready to go

**Pull request merged**